mlir/include/mlir/Dialect/OpenMP/OpenMPOps.td

```diff
@@ -187,8 +188,10 @@ def ParallelOp : OpenMP_Op<"parallel", [
              Variadic<AnyType>:$allocate_vars,
              Variadic<AnyType>:$allocators_vars,
              Variadic<OpenMP_PointerLikeType>:$reduction_vars,
+             Variadic<AnyType>:$private_vars,
```

Comment:
Is it possible to move `private_vars` to be right before `private_inits`?

Reply:
Done.